server.go

```diff
@@ -141,16 +134,9 @@ func (s *Server) WithTLS(tlsConfig *tls.Config) *Server {
 	return s
 }
 
-// WithErrorLogger sets the logger to use for error logging.
-func (s *Server) WithErrorLogger(f LogFunc) *Server {
-	s.errorLog = f
-
-	return s
-}
-
-// WithDebugLogger sets the logger to use for debug logging.
-func (s *Server) WithDebugLogger(f LogFunc) *Server {
-	s.debugLog = f
+// WithLogger sets the logger to use for error and debug logging.
```

Comment:
Perhaps some better documentation. By default we will log all headers and properties, especially on errors so the user must ensure that the logger has a handler that redacts any sensitive information.